src/Sylius/Bundle/ApiBundle/Command/Admin/RequestResetPasswordEmail.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Admin;
+
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
+
+/** @experimental */
+class RequestResetPasswordEmail implements IriToIdentifierConversionAwareInterface
+{
+    public function __construct(private string $email)
```

Comment:
we need to unify our approach to commands. For now, in most cases we were using public properties, unless data comes from data enrichment

```suggestion
    public function __construct(public string $email)
```